.github/workflows/nexus-publish.yml

```
@@ -0,0 +1,42 @@
+# This workflow will build a package using Maven and then publish it to
+# qbic-repo.qbic.uni-tuebingen.de packages when a release is created
+# For more information see: https://github.com/actions/setup-java#apache-maven-with-a-settings-path
+
+name: Nexus Package
+
+on:
+  push:
+    branches:
+      - development
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
```

I think we should keep the JDK version consistent for all github actions 
```suggestion
          java-version: 1.8
```